puppet/modules/profiles/manifests/discourse.pp

```diff
@@ -2,8 +2,25 @@
 # @see https://github.com/discourse/discourse/blob/main/docs/INSTALL-cloud.md
 class profiles::discourse {
   $root = '/var/discourse'
```

Comment:
I'd prefer to use:
```puppet
if $facts['os']['family'] == 'RedHat' {
  include discourse
  $root = $discourse::root
} else {
  $root = '/var/discourse'
}
```

You can also explicitly pass the root to the class, but I'm not sure that adds too much value.

Come to think of it, if we're templating the containers directory, does it really make sense to back it up? Perhaps:

```puppet
if $facts['os']['family'] == 'RedHat' {
  include discourse
  $root = $discourse::root
  $backup_path = ["${discourse::root}/shared/standalone/backups"]
} else {
  $root = '/var/discourse'
  $backup_path = ["${root}/containers", "${root}/shared/standalone/backups"]
}
```